docs/css/overrides.min.css

```diff
@@ -1,3 +1,156 @@
+.md-typeset .tx-insiders {
```

Comment:
File is not minified anymore as the name `overrides.min.css` suggests.
Should we rename it or edit the none minified version? Or is this not supported by the framework?

Reply:
fixed